llvm/lib/LTO/LTO.cpp

```diff
@@ -1443,6 +1462,23 @@ class InProcessThinBackend : public ThinBackendProc {
       CfiFunctionDecls.insert(
           GlobalValue::getGUID(GlobalValue::dropLLVMManglingEscape(Name)));
   }
+};
+
+class InProcessThinBackend : public CGThinBackend {
```

Comment:
Can you add a comment to this class for completeness?

Reply:
Done. Thanks.